alerts.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  alert_resource_group_name = "disposer-shared-${var.env}"
+}
+
+module "idam-user-disposer-action-group" {
+  source                 = "git@github.com:hmcts/cnp-module-action-group"
+  location               = "global"
+  env                    = var.env
+  resourcegroup_name     = local.alert_resource_group_name
+  action_group_name      = "${var.application_name}-${var.env}-ag"
+  short_name             = "dispr-alert"
+  email_receiver_name    = "Idam User Disposer Failure Alert"
+  email_receiver_address = data.azurerm_key_vault_secret.idamUserDisposerAlertEmail.value
+}
+
+module "idam-user-disposer-run-failure-alert" {
+  source                     = "git@github.com:hmcts/cnp-module-metric-alert"
+  location                   = var.location
+  app_insights_name          = "disposer-${var.env}"
+  alert_name                 = "${var.application_name}-${var.env}-run-failure"
+  alert_desc                 = "Alert when idam user disposer fail to run"
+  app_insights_query         = "traces | where message contains 'Disposer Idam User Summary :'"
```

Comment:
Isn't it should be does not contain 'Disposer Idam user Summary :"

Reply:
I am looking for that specific message and then checking if the number of matching messages is 0, raise an alert. At least that is the way I understand.